CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.9 FATAL_ERROR)
+cmake_minimum_required(VERSION 3.20 FATAL_ERROR)
```

Comment:
What is the default CMake version on the oldest distribution we support?

Reply:
It's 3.26.5 on `alma8`, so we're safe there. In any case, we don't have a choice I think because the minimum CMake version 3.20 is inherited from LLVM that we depend on as a builtin.